### PR TITLE
Fixes a buildmode runtime

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: code/modules/admin/verbs/buildmode.dm [hunk 1]
```diff
@@ -293,7 +293,7 @@
 					var/turf/T = get_turf(object)
 					log_admin("Build Mode: [key_name(user)] modified [T] ([T.x],[T.y],[T.z]) to [objholder]")
 					T.ChangeTurf(objholder)
-				else
+				else if(objholder)
```
Comment (optimumtact) [inline code/modules/admin/verbs/buildmode.dm]: is it valid for it to be null at this point or is the bug that the objholder is null

Comment (Cyberboss) [inline code/modules/admin/verbs/buildmode.dm]: Giff +BUILDMODE and I'll tell ;)

Seriously though, I assume it's due to clicking in buildmode with no object selected

**Comment by PKPenguin321** [comment]
If you need to test buildmode on a live server, seek any admin with AdminTrainer or higher and ask for a temp promotion

**Comment by PKPenguin321** [comment]
Seek them out ingame, I mean

File: code/modules/admin/verbs/buildmode.dm (hunk 1 above)
Comment (lzimann) [inline code/modules/admin/verbs/buildmode.dm]: [Unless I am missing something, this should not be null](https://github.com/tgstation/tgstation/blob/ca32b8efba568139b2e3c779caf5c72e93cd2da0/code/modules/admin/verbs/buildmode.dm#L165)

**Comment by ChangelingRain** [comment]
This isn't a proper fix, as, indeed, objholder should not be null.

**Pull request closed without merging**